x-pack/plugins/cases/public/components/user_actions/assignees.tsx

```diff
@@ -5,13 +5,165 @@
  * 2.0.
  */
 
-import type { UserActionBuilder } from './types';
+import { EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+import { UserProfileWithAvatar } from '@kbn/user-profile-components';
+import React, { memo } from 'react';
+import { SnakeToCamelCase } from '../../../common/types';
+import { Actions, AssigneesUserAction, User } from '../../../common/api';
+import { getName } from '../user_profiles/display_name';
+import { Assignee } from '../user_profiles/types';
+import { UserToolTip } from '../user_profiles/user_tooltip';
+import { createCommonUpdateUserActionBuilder } from './common';
+import type { UserActionBuilder, UserActionResponse } from './types';
+import * as i18n from './translations';
+import { getUsernameDataTestSubj } from '../user_profiles/data_test_subject';
+
+const FormatListItem: React.FC<{
```

Comment:
A comma is placed if there are two people assigned. I think it should not have a comma. Like: `<name1> and <name2>` and not `<name1>, and <name2>`

<img width="561" alt="Screenshot 2022-08-30 at 5 58 08 PM" src="https://user-images.githubusercontent.com/7871006/187473834-8771ab85-5512-4f74-9aec-914e7065c26f.png">

Reply:
Great catch!